### PR TITLE
Add support for Hub V2 modules conversion.

### PR DESCRIPTION
**Pull request by vbardiovskyg**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-converter/383)

### CODE REVIEW

File: python/tensorflowjs/converters/tf_saved_model_conversion_v2.py [hunk 1]
```diff
@@ -391,3 +405,43 @@ def convert_tf_hub_module(module_path, output_dir,
     # Clean up the temp files.
     if os.path.exists(frozen_file):
       os.remove(frozen_file)
+
+
+def convert_tf_hub_module(module_handle, output_dir,
+                          signature='default', saved_model_tags='',
```
Comment (pyu10055) [inline python/tensorflowjs/converters/tf_saved_model_conversion_v2.py]: should this have a default value?

Comment (vbardiovskyg) [reply python/tensorflowjs/converters/tf_saved_model_conversion_v2.py]: There is actually no reasonable default, but will set it to 'serve' for now.

**Comment by vbardiovskyg** [comment]
Answering the high level question: the code paths I am adding here have no special dependency on nightly vs TF1.14.

(This of course assumes that you will migrate from tf.saved_model.load to tf.saved_model.load_v2, which will be the proper symbol to use in TF1.X)

**Comment by vbardiovskyg** [comment]
Thanks for the review!

**Comment by pyu10055** [comment]
> Answering the high level question: the code paths I am adding here have no special dependency on nightly vs TF1.14.
> 
> (This of course assumes that you will migrate from tf.saved_model.load to tf.saved_model.load_v2, which will be the proper symbol to use in TF1.X)

Cool, we have migrated to 1.14, and since we are importing the file directly, there is not change to the load method.

**Comment by pyu10055** [comment]
> > Answering the high level question: the code paths I am adding here have no special dependency on nightly vs TF1.14.
> > (This of course assumes that you will migrate from tf.saved_model.load to tf.saved_model.load_v2, which will be the proper symbol to use in TF1.X)
> 
> Cool, we have migrated to 1.14, and since we are importing the file directly, there is not change to the load method.

Since you are publish 5.0 next week, it might be better to wait for that. We are trying not to include nightly in the deps, since all the others are removed.

**Pull request merged**